packages/manager/modules/cloud-connect/src/overview/overview.routing.js

```diff
@@ -0,0 +1,110 @@
+        $state,
+        cloudConnect,
+      ) => (interfaceId) =>
+        $state.go('cloud-connect.overview.remove-pop', {
+          interfaceId,
+          popId: cloudConnect.getPopConfiguration(interfaceId).id,
+        }),
+      goToLockPortPage: /* @ngInject */ ($state) => (interfaceId) =>
+        $state.go('cloud-connect.overview.lock-port', { interfaceId }),
+      goToUnlockPortPage: /* @ngInject */ ($state) => (interfaceId) =>
+        $state.go('cloud-connect.overview.unlock-port', { interfaceId }),
+      goToSendServiceKeyPage: /* @ngInject */ ($state, cloudConnect) => () =>
+        $state.go('cloud-connect.overview.send-service-key', {
+          serviceKeyId: cloudConnect.getActiveServiceKey().id,
+        }),
+      goToDatacenterAddPage: /* @ngInject */ ($state) => () =>
+        $state.go('cloud-connect.overview.datacenter-add'),
+      goToDatacenterAddExtraPage: /* @ngInject */ ($state) => (datacenterId) =>
+        $state.go('cloud-connect.overview.datacenter-add-extra', {
+          datacenterId,
+        }),
+      goToRemoveDatacenterConfigurationPage: /* @ngInject */ ($state) => (
+        datacenterId,
+      ) =>
+        $state.go('cloud-connect.overview.remove-datacenter-configuration', {
+          datacenterId,
+        }),
+      goToRemoveExtraPage: /* @ngInject */ ($state) => (
+        datacenterId,
+        extraId,
+      ) =>
+        $state.go('cloud-connect.overview.remove-extra', {
+          datacenterId,
+          extraId,
+        }),
+      goToManageServiceKeysPage: /* @ngInject */ ($state) => () =>
+        $state.go('cloud-connect.service-keys'),
+      tasksHref: /* @ngInject */ ($state, cloudConnect) =>
+        $state.href('cloud-connect.tasks', {
+          ovhCloudConnectId: cloudConnect.id,
+        }),
+      goToCloudConnectPage: /* @ngInject */ (
+        $state,
+        $translate,
+        CucCloudMessage,
+        CucControllerHelper,
+        cloudConnectId,
+        cloudConnectService,
+      ) => (message = false, type = 'success', reload = false, vrackId) => {
+        const state = 'cloud-connect.overview';
+        const promise = $state.go(
+          state,
+          {
+            ovhCloudConnectId: cloudConnectId,
+          },
+          {
+            reload,
+          },
+        );
+        if (message) {
+          promise.then(() => {
+            if (vrackId) {
+              cloudConnectService
+                .getVrackAssociatedCloudConnect(vrackId)
+                .then((res) => {
+                  if (res) {
+                    CucCloudMessage.warning(
+                      $translate.instant(
+                        'cloud_connect_vrack_associate_cloud_connect',
+                      ),
+                      state,
+                    );
+                  }
+                });
+            }
+            CucCloudMessage[type](message, state);
+            CucControllerHelper.scrollPageToTop();
+          });
+        }
+        return promise;
```

Comment:
**suggestion** You should do the test `if (message)` directly in `then` block, to avoid confusion